### PR TITLE
Add codespaces support

### PR DESCRIPTION
**Pull request by electrocucaracha**
### Describe your changes
[GitHub Codespaces](https://docs.github.com/en/codespaces/overview) provides an easy way to deploy remote development environments. This change customize the setup process for deploying development environments.

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by iamNoah1** [comment]
cc @jihoon-seo @seokho-son what do you think about this?

**Comment by jihoon-seo** [comment]
I think that this will be useful for those who want to handle the Glossary jobs on their own GitHub codespace instance.
LGTM 😊

File: README.md [hunk 1]
```diff
@@ -33,3 +33,5 @@ npm install
 ```
 
 You can then run the site using `hugo server`.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=329970318)
```
Comment (seokho-son) [inline README.md:37]:
```suggestion
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?repo=cncf/glossary)
```
I think we don't have to hide repo_select and would better to show repository name directly.

**Review by seokho-son — APPROVED** [review]
Hi @electrocucaracha 
I think providing codespace badge is good for new contributors. Thanks for this suggestion!
Maybe we'd better to add some information later regarding how to contribute by using the codespace. (contributors need to use appropriate branches for fetching, pushing, ..)

I have a minor comment which can be applied directly.

**Comment by seokho-son** [comment]
Hi @jihoon-seo 
This PR needs one more approval.

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Pull request merged**